wire-ios-data-model/Source/Model/FeatureConfig/FeatureService.swift

```diff
@@ -230,6 +230,29 @@ public class FeatureService {
         }
     }
 
+    // MARK: - MLS Migration
+
+    public func fetchMLSMigration() -> Feature.MLSMigration {
+        guard
+            let feature = Feature.fetch(name: .mlsMigration, context: context),
+            let featureConfig = feature.config
+        else {
+            return .init()
+        }
+
+        let config = try! JSONDecoder().decode(Feature.MLSMigration.Config.self, from: featureConfig)
+        return .init(status: feature.status, config: config)
+    }
+
+    public func storeMLSMigration(_ mlsMigration: Feature.MLSMigration) {
+        let config = try! JSONEncoder().encode(mlsMigration.config)
```

Comment:
same question as above

Reply:
I removed the force try, returning early in case of failure.